scripts/cbmc_json_parser.py

```diff
@@ -57,24 +57,28 @@ def main():
     """
     Script main function.
     Usage:
-      > cbmc_json_parser.py <cbmc_output.json> [<format>]
+      > cbmc_json_parser.py <cbmc_output.json> <format> [--extra-ptr-check]
     """
     # Check only one json file as input
-    if len(sys.argv) < 2:
-        print("Json File Input Missing")
+    if len(sys.argv) < 3:
+        print("Error: Missing required arguments. Usage:")
+        print("cbmc_json_parser.py <cbmc_output.json> <format> [--extra-ptr-check]")
         sys.exit(1)
 
-    if len(sys.argv) == 3:
-        output_style = output_style_switcher[sys.argv[2]]
-    else:
-        output_style = "regular"
+    output_style = output_style_switcher[sys.argv[2]]
+    if not output_style:
+        print(f"Error: Invalid output format {sys.argv[2]}. Usage:")
+        print("cbmc_json_parser.py <cbmc_output.json> <format> [--extra-ptr-check]")
+
+    extra_ptr_check = len(sys.argv) == 4 and sys.argv[3] == "--extra-ptr-check"
```

Comment:
Not sure I understand.

Reply:
I meant, if the script is called with four arguments, but the 4th one (`sys.argv[3]`) is not `--extra-ptr-check` (e.g. due to a typo), the script will silently ignore it. Not a big deal since the script is meant to be called by `cargo kani`, so it can be done later.

Reply:
Ah. That makes sense. I can do that. Thanks